opentreemap/treemap/templates/treemap/partials/photo_carousel.html

```diff
@@ -46,5 +46,5 @@
 <a class="carousel-control right" href="#photo-carousel" data-slide="next"><i class="icon-right-circled"></i></a>
 {% endif %}
 {% if error %}
-<div class="text-danger">{{ error }}</div>
+<div class="alert alert-danger js-image-upload-error">{{ error }}</div>
```

Comment:
Should the class here be `js-image-upload-error` or `js-upload-error`? Other places in this diff I see us deleting markup with the `js-image-upload-error` class. When I grep the codebase for `js-image-upload-error` this is the only reference I find.